pkgs/development/ruby-modules/with-packages/test.nix

```diff
@@ -0,0 +1,94 @@
+# a generic test suite for all gems for all ruby versions.
+# use via nix-build.
+let
+  pkgs = import ../../../.. {};
+  lib = pkgs.lib;
+  stdenv = pkgs.stdenv;
+
+  rubyVersions = with pkgs; [
+    # ruby_2_4
+    ruby_2_5
+    # ruby_2_6
+  ];
+
+  gemTests = (builtins.mapAttrs (name: gem:
+    ''
+      require "${name}"
+    ''
+  ) pkgs.ruby.gems) // {
+    actioncable = "require 'action_cable'";
+    actionmailer = "require 'action_mailer'";
```

Comment:
Is this a list of gems with paths that don’t match their names? If so, since there are so many, I think we should drop the `require “${name}”` case and just list them all.

Reply:
I'd rather not list them all, since it's 194 gems, and the exceptions are just 41 of them. But I'll use arrays of require names to make it easier to read :)